src/component/Cards/NFTCard.jsx

```diff
@@ -0,0 +1,18 @@
+function NFTCard({ content, isAvailable }) {
+  const availabilty = isAvailable ? "Available" : "Not Available";
```

Comment:
This variable isn't required

Reply:
I forgot to use the variable. The idea was to use the isAvailabel boolean to render the text below the nft signaling its availability.